starboard/linux/x64x11/clang/3.9/platform_configuration/configuration.gni

```diff
@@ -12,38 +12,48 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import("//starboard/build/config/base_configuration.gni")
+if (current_toolchain == default_toolchain &&
+    build_with_separate_cobalt_toolchain) {
+  import("//starboard/evergreen/x64/platform_configuration/configuration.gni")
+  platform_tests_path =
+      "//starboard/linux/shared:starboard_platform_tests($starboard_toolchain)"
+  cobalt_font_package = "standard"
 
-nasm_exists = true
+  sb_has_unused_symbol_issue = true
```

Comment:
Is this something we want to set only for the cobalt toolchain? My only thought is that someone might try to use this in the starboard toolchain and be surprised with how it's set, but maybe we don't want to use it for targets being built by the starboard toolchain at all?

Also remember to make the playstation change internally first

Reply:
The issue showed up only with the cobalt toolchain. Could be related to the stripping unused symbols when building a shared library. I would prefer to keep the scope narrow and match the observed issues.